mmseg/datasets/transforms/transforms.py

```diff
@@ -1226,3 +1226,156 @@ def __repr__(self):
         repr_str += f'edge_width={self.edge_width}, '
         repr_str += f'ignore_index={self.ignore_index})'
         return repr_str
+
+
+@TRANSFORMS.register_module()
+class MedicalRandomFlip(BaseTransform):
+    """Reverse the orders of elements in an 3D Medical image & gt_seg_map along
+    a given axe.
+
+    Required Keys:
+
+    - img
+    - gt_seg_map
+
+    Modified Keys:
+
+    - img
+    - gt_seg_map
+
+    Added Keys:
+
+    - flip
+    - flip_direction
+    - swap_seg_labels (optional)
+
+    Args:
+        prob (float | list[float]): The flipping probability. Defaults to None.
+        direction (int, list[int]): The flipping direction (Spatial axes along
+            which to flip over).
+        swap_seg_labels (list, optional): The label pair need to be swapped
+            for ground truth, like 'left arm' and 'right arm' need to be
+            swapped after horizontal flipping. For example, ``[(1, 5)]``,
+            where 1/5 is the label of the left/right arm. Defaults to None.
+
+        If the input is a list, the length must equal len(prob), each element
```

Comment:
which is the input? prob, direction, or swap seg labels?   the description must follow the argument.